OpenRA.Mods.Common/Widgets/Logic/Editor/ActorEditLogic.cs

```diff
@@ -72,8 +72,8 @@ public ActorEditLogic(Widget widget, World world, WorldRenderer worldRenderer, D
 			editor = widget.Parent.Parent.Get<EditorViewportControllerWidget>("MAP_EDITOR");
 			editor.DefaultBrush.SelectionChanged += HandleSelectionChanged;
 
-			var selectTabContainer = widget.Parent.Parent.Get<ContainerWidget>("SELECT_WIDGETS");
-			actorEditPanel = selectTabContainer.Get<ContainerWidget>("ACTOR_EDIT_PANEL");
+			var selectTabContainer = widget.Parent.Parent.Get<Widget>("SELECT_WIDGETS");
```

Comment:
`Widget` is the default, so can be omitted, like many of the other .Gets above and below(!)

Reply:
omitted